src/normalize.js

```diff
@@ -4,11 +4,12 @@ const Contentstack = require("contentstack");
 const { ACTIVE_ENV, NODE_ENV } = process.env;
 const activeEnv = ACTIVE_ENV || NODE_ENV || "development";
 require("dotenv").config({ path: `.env.${activeEnv}` });
-
 const apiKey = process.env.CONTENTSTACK_API_KEY;
 const apiToken = process.env.CONTENTSTACK_ACCESS_TOKEN;
 const environment = process.env.CONTENTSTACK_ENVIRONMENT;
+const defaultLocale = process.env.CONTENTSTACK_DEFAULT_LOCALE || "en-us";
 const Stack = Contentstack.Stack(apiKey, apiToken, environment);
+const { region = "en" } = require(`${process.cwd()}/package.json`);
```

Comment:
Do we need the default value? If so, make sure it's a region rather than a language.